test/ANNOTATIONS.yaml

```diff
@@ -207,6 +207,8 @@ all:
     - Stack allocate on and task function argument bundles (#4074)
   12/08/16:
     - Re-enable denormalization (#4547)
+  12/12/16:
```

Comment:
I think this one should be 12/13/16 instead.  Multilocale testing worked that night and so the change turns up there, and there was a different set of tests impacted by commits from the 13th.

At least, that's only if I'm right in remember that we enabled annotations on days there isn't data (which I thought I saw go by a while ago, but I may be misremembering)